Solutions/Authomize/Analytic Rules/New_service_account_gained_access_to_IaaS_resource.yaml

```diff
@@ -0,0 +1,59 @@
+id: 6c17f270-cd56-48cc-9196-1728ffea6538
+name: New service account gained access to IaaS resource
```

Comment:
@sriley0975 As Devika, mentioned in one of her comment, you could also use dynamic variable to include all your 'Policy' name and also Dynamic alert name and description. All these detections can potentially be handled by this one rule.

Reply:
checkout line no. 20 and 27 for use of 'dynamic' function

Reply:
I've gone through this - not sure how this might work with the different events generated and coming in  - I'll have to play with this a bit and see if it will do what should be done with the data generated. At the moment I'll finish with the ones I have - then look into this for next version?

Reply:
we do have many names as in 
szOperationNames
But does teh description and tactics change? As we add more detail what happens ie when we create additional policies and incidents - hence I want to play with this a bit - but meanwhile I we need to drive to completion of this initial release. Hope that makes sense.

Reply:
Hmm - Are you saying write a query more general and just capture anything within a "period" of time and create incidents based on that? This would then be dynamic and create incidents whne they are seen?

Reply:
OK - did some reasearch - I can do this - but its going to take more time - I would like to move forward with what we have and I'll concentrate and doing this updates - SO if we get to an initial release we have something to point at - But I will work to understand and build out this - I'm seeing that I can use unctions such as dynamic, which can create a dynamic data type object (like the array szOperationNames), or make-series - these will allow me to bubble the information up to do things such as 
alertDetailsOverride:
  alertnameFormat: Alert from Authomize - {AlertName}
  alertDescriptionFormat: |
    {AlertDescription}